### PR TITLE
Update bending modes and remove ZCS from ts_ofc

### PR DESCRIPTION
**Pull request by gmegh**
This branch tackles the following points

- Updates the sensitivity matrix and intrinsic zernikes to latest version of batoid and batoid_rubin, using the new uncorrupted bending modes
- Removes the Zemax Coordinate System (ZCS) from ts_ofc. This implies that we do everything in the Camera Coordinate System (CCS). The sample points are updated to fix the x direction sign flip.
- Updates the test data to match the new coordinate system and new bending modes
- Removes old unused test data

### CODE REVIEW

File: doc/versionHistory.rst [hunk 1]
```diff
@@ -6,6 +6,16 @@
...
 
+.. _lsst.ts.ofc-3.0.1:
+
+v3.0.1
+======
+
+* Removed Zemax Coordinate System, which is substituted by the Camera Coordinate System. 
+* Updates the bending modes used to the new uncorrupted bending modes.
+* Updates sensitivity matrix and intrinsic zernikes for the new coordinate system and new bending modes.
+* Removes old unused test data.
```
Comment (teweitsai) [inline doc/versionHistory.rst:17]: Please help to use the past tense to align with others.

Updates --> Updated
Removes --> Removed

File: python/lsst/ts/ofc/scripts/generate_sensitivity_matrix.py [hunk 2]
```diff
@@ -345,7 +336,7 @@ def main(args: argparse.Namespace) -> None:
     """
 
     # Load configuration
-    config_dir = pathlib.Path(__file__).resolve().parents[1] / "policy"
+    config_dir = pathlib.Path(__file__).resolve().parents[5] / "policy"
```
Comment (teweitsai) [inline python/lsst/ts/ofc/scripts/generate_sensitivity_matrix.py:339]: I think you could use the `get_config_dir()` in `utils.py` or no?

File: python/lsst/ts/ofc/scripts/generate_intrinsic_zernikes.py [hunk 3]
```diff
@@ -104,7 +104,7 @@ def main(args: argparse.Namespace) -> None:
     """
 
     # Load configuration file
-    config_dir = pathlib.Path(__file__).resolve().parents[1] / "policy"
+    config_dir = pathlib.Path(__file__).resolve().parents[5] / "policy"
```
Comment (teweitsai) [inline python/lsst/ts/ofc/scripts/generate_intrinsic_zernikes.py:107]: I think you could use the `get_config_dir()` in `utils.py` or no?

File: tests/testData/lsst_wfs_error_iter0.z4c [hunk 4]
```diff
@@ -3,7 +3,7 @@
...
+-3.77046951e-01  2.46893933e-01  1.71913068e-01 -1.07190906e-01 -9.65822227e-03  1.33641855e-01 -3.54920415e-02 -9.33135598e-02  5.32257498e-01  2.87757909e-01  4.91834594e-02 -8.66900648e-03  -2.02145722e-01 -5.12041641e-01  1.03463883e-03 -1.00569797e-01 -7.63394019e-04  3.80833466e-02  4.40000646e-02
+3.68729853e-01  7.19871261e-02 -1.73366543e-01 -1.41318159e-01  2.54360313e-02 -5.91249702e-02 -2.35364232e-01  1.02515612e-02 -5.30917456e-01 -2.16012925e-01  3.61699513e-02  1.19505532e-02  -2.46373380e-01 -4.66946875e-01  1.32427611e-01  2.74183131e-02  2.51153234e-02  1.12182806e-02 -5.56776478e-02
+-7.55256690e-01 -2.72719521e-01  7.03935362e-02 -1.32953238e-01 -6.65246084e-02  2.24357821e-02  6.35896646e-02 -1.37400911e-01  2.53307204e-01 -4.79416182e-01 -1.16716811e-02  5.09423086e-03  -1.07825965e-01 -4.47606628e-01 -8.04652829e-02  3.16979902e-02 -2.92085226e-03  1.31844115e-02  9.31604917e-02
```
Comment (teweitsai) [inline tests/testData/lsst_wfs_error_iter0.z4c:9]: Please add a new line in the end of file. This comment applies to other similar/related files.

File: tests/test_sensitivity_matrix.py [hunk 5]
```diff
@@ -65,19 +65,67 @@ def setUp(self):
             # which we have excluded from the unrotated_sensitivity_matrix.
             self.gq_sensitivity_matrix = self.gq_sensitivity_matrix[:31, ...]
 
+            # Because the old sensitivity matrix was in the Zemax Coordinate
+            # System we need to swap the following zernikes:
```
Comment (teweitsai) [inline tests/test_sensitivity_matrix.py:69]: I thought you meant the sensitivity matrix had been updated to have the consistent coordinate system as others at the meeting at the last week?

If this is not done yet, do you have another ticket to do so? If yes, please help to put the ticket number here. We would need to have a plan to unify the coordinate system for all the policy files in this repository to make the future's maintenance easier.

File: tests/test_sensitivity_matrix.py [hunk 6]
```diff
@@ -65,19 +65,67 @@ def setUp(self):
...
+            swap = np.array([5, 8, 10, 13, 15, 16, 18, 20])
+            self.gq_sensitivity_matrix[:, swap - self.ofc_data.znmin] *= -1
+
+            # Similarly, we need to swap the following gaussian
+            # quadrature points because the old sensitivity matrix
+            # was in the Zemax Coordinate System and therefore
+            # there is a flip in the x direction. So the following
+            # swap, flips those GQ points that have opposite x sign.
```
Comment (teweitsai) [inline tests/test_sensitivity_matrix.py:77]: Same comment of the coordinate system above.

**Review by teweitsai — APPROVED** [review]
Looks great to me to simplify the coordinate system transformation!

File: tests/test_sensitivity_matrix.py (hunk 5 above)
Comment (gmegh) [reply tests/test_sensitivity_matrix.py:69]: That's right. This swap here is performed to check it against the old sensitivity matrix which did not have the same coordinate system. Therefore the old sensitivity matrix needs to be swaped to match the new one. We could also remove this test, but I think it's useful.

File: tests/test_sensitivity_matrix.py (hunk 6 above)
Comment (gmegh) [reply tests/test_sensitivity_matrix.py:77]: Same as above

File: python/lsst/ts/ofc/state_estimator.py [hunk 7]
```diff
@@ -105,7 +105,7 @@ def dof_state(
 
         # Evaluate sensitivity matrix at sensor positions
         sensitivity_matrix = self.dz_sensitivity_matrix.evaluate(
-            field_angles, rotation_angle
+            field_angles, -rotation_angle
```
Comment (teweitsai) [inline python/lsst/ts/ofc/state_estimator.py:108]: Please add the comment to explain this sign change. Thanks!

**Review by teweitsai — APPROVED** [review]
Looks good.

File: python/lsst/ts/ofc/state_estimator.py (hunk 7 above)
Comment (gmegh) [reply python/lsst/ts/ofc/state_estimator.py:108]: With the change of the coordinate systems in the XY plane, we need to ensure that the rotation of the Zouble Zernike object in the UV plane is opposite to that one.

Comment (teweitsai) [reply python/lsst/ts/ofc/state_estimator.py:108]: Please put the above comment in the code. Thanks!

**Pull request merged**